packages/flutter/lib/src/widgets/tooltip.dart

```diff
@@ -0,0 +1,910 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// @docImport 'package:flutter/material.dart';
+///
+/// @docImport 'gesture_detector.dart';
+library;
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+
+import 'basic.dart';
+import 'debug.dart';
+import 'feedback.dart';
+import 'framework.dart';
+import 'media_query.dart';
+import 'overlay.dart';
+import 'selection_container.dart';
+import 'ticker_provider.dart';
+
+const AnimationStyle _kDefaultAnimationStyle = AnimationStyle(
+  curve: Curves.fastOutSlowIn,
+  duration: Duration(milliseconds: 150),
+  reverseDuration: Duration(milliseconds: 75),
+);
+
+/// Signature for building the tooltip overlay child.
+///
+/// The `animation` parameter is an [Animation] that maps to the tooltip's show
```

Comment:
Do you think it'd be better to move the detailed explanation to the class property (`RawTooltip.tooltipBuilder`), i.e. the function signature only has the first paragraph? This is because developers will probably only come it from `RawTooltip.tooltipBuilder` anyway.

Also link the method where this signature is used.